guides/common/modules/ref_best-practices-for-content-views.adoc

```diff
@@ -0,0 +1,19 @@
+[id="best-practices-for-content-views_{context}"]
+= Best practices for content views
+
+* Create content views that bundle content for {client-os} and additional software such as `Apache-2.4` or `PostgreSQL-16.2`.
+Content views that are too small require more effort to maintain.
+* If you require daily updated content, use the content view `Default Organization View`, which contains the latest synchronized content from all repositories and is available in the _Library_ lifecycle environment.
+* Restrict composite content views to situations that require greater flexibility, for example, if you update one content view on a weekly basis and another content view on a monthly basis.
+* If you use composite content views, first publish the content views and then publish the composite content views.
+Bundling content views involves a trade-off.
+The more content views you bundle, the more effort is needed to change or update content.
+Make it as simple as possible and as complex as necessary.
```

Comment:
```suggestion
```
This doesn't really tell the user how to make it simple or complex. Seems like a filler sentence. Not sure it is needed.